### PR TITLE
Createroute

### PR DESCRIPTION
**Pull request by Hacene010**

### CODE REVIEW

**Review by GeorgesFORTRY — APPROVED** [review]
Good job ! 👍

File: index.js [hunk 1]
```diff
@@ -1,15 +1,57 @@
...
+        const row = rows[i];
+        const myKey = row.tagname;
+        const myValue = {
+          src: row.img_src,
+          alt: row.img_alt,
+        };
+        results[myKey] = myValue;
+      }
+
+      console.log(results);
```
Comment (LoicBrassart) [inline index.js]: No `console.log` in production

File: data/db.sql [hunk 2]
```diff
@@ -0,0 +1,185 @@
+-- MySQL dump 10.13  Distrib 8.0.25, for Linux (x86_64)
```
Comment (LoicBrassart) [inline data/db.sql:1]: The vuerd file generated by the VSC plugin generates a clunky database structure, I'll recreate it myself today. For now it's okay, but we'll replace this file quickly

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One silly evil

**Review by GeorgesFORTRY — APPROVED** [review]
it's ok despite additional changes

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   "author": "",
   "license": "ISC",
   "scripts": {
-    "start": "node index.js",
+    "start": "nodemon index.js",
```
Comment (LoicBrassart) [inline package.json]: Oh noes, absolutely not! >_<
Why: `nodemon` is a dev dependancy, but the `npm run start` will be executed on the production machine (where `nodemon` will be absent), you should not do this
Alternative: you could create a "dev" script to run the website with `nodemon` if you want

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One more Evil

**Review by GeorgesFORTRY — APPROVED** [review]
ok !

**Review by LoicBrassart — APPROVED** [review]
No more Evils!

**Pull request merged**